db/model/generator.js

```diff
@@ -166,16 +165,18 @@ module.exports = function generator(seq, dataTypes) {
         const gtVersion = inst.generatorTemplate.version;
         if (inst.changed('generatorTemplate') || inst.changed('context')) {
           return seq.models.GeneratorTemplate.getSemverMatch(gtName, gtVersion)
-            .then((ret) => {
-              if (!ret) {
-                throw new ValidationError('No Generator Template matches ' +
-                `name: ${gtName} and version: ${gtVersion}`);
+            .then((gt) => {
+              if (!gt) {
+                throw new ValidationError('No Generator Template matches' +
```

Comment:
always prefer space at the end of the line rather than at the beginning of the next line